testbed/testbed/utils.go

```diff
@@ -23,3 +23,10 @@ import (
 func GetAvailablePort(t *testing.T) int {
 	return int(testutils.GetAvailablePort(t))
 }
+
+type ProcConfiguration struct {
```

Comment:
`Proc` is a bit unclear abbreviation. Let's use `Processor`. `Configuration` can be safely abbreviated to `Config`, as it is commonly done in the codebase, so `ProcessorConfig` should be fine.

Reply:
Also add a common on what this struct is for.